### PR TITLE
Chapter 4 Rails-flavored Ruby

### PR DESCRIPTION
**Pull request by leducanh120197**

### CODE REVIEW

**Comment by leducanh120197** [comment]
ready

**Comment by leducanh120197** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield :title  %> | <%= yield :name %></title>
+    <title><%= full_title yield :title %></title>
```
Comment (DuongNT1512) [inline app/views/layouts/application.html.erb]:
```suggestion
    <title><%= full_title yield(:title) %></title>
```

**Comment by leducanh120197** [comment]
ready

**Comment by DuongNT1512** [comment]
merged

**Pull request merged**